### ISSUE TITLE
Kubernetes version 1.15.12 no longer supported and dashboard being deprecated

### ISSUE THREAD

**Issue by waeltken** [microsoft/MCW-Cloud-native-applications#176]
AKS no longer supports kubernetes 1.15 and the hands-on-lab needs an update. Later kubernetes versions on AKS deprecate the dashboard requiring the hands-on-lab instructions to be updated.

**Comment by timahenning** [comment]
@waeltken Thank you for the heads up.  We will try to get this soon.

**Comment by waeltken** [comment]
We have people delivering the workshops on 28.09. and 30.09..

My suggestion would be to install the k8s dashboard manually, generate an admin user and use `kubectl proxy` to connect to the dashboard. With this the lab can be completed as described.
```
kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/v2.0.4/aio/deploy/recommended.yaml
```

https://github.com/kubernetes/dashboard/blob/master/docs/user/access-control/creating-sample-user.md#creating-a-service-account

**Comment by DawnmarieDesJardins** [comment]
Closing issue as resolved with merge of PR #177
